src/utils/resizeObserver.js

```diff
@@ -0,0 +1,9 @@
+export default class ResizeObserver {
+  observe() {
+    // do nothing
+  }
+
+  unobserve() {
+    // do nothing
+  }
+}
```

Comment:
Remove this code, please, and create a mock for the `ResizeObserver` object inside your test